airbyte-integrations/connectors/source-zendesk-support/unit_tests/unit_test.py

```diff
@@ -243,20 +243,6 @@ def test_check_start_time_param():
     assert output == expected
 
 
-@pytest.mark.parametrize(
```

Comment:
Some cleanup here but I didn't remove the import because we still have TicketMetrics which uses the Python implementation of Tickets. My guess is that this whole file will just be deleted soon anyway